pkg/test/env.go

```diff
@@ -245,9 +253,7 @@ func StartControlPlane(port string, installCRDs, installWebhooks bool) (*rest.Co
 	var kubeConfig []byte
 	// we extract the kubeconfig from env var if we are using an existing cluster
 	if useExistingGreenhouseCluster {
-		kubeConfigLocation := os.Getenv("KUBECONFIG")
-		Expect(kubeConfigLocation).NotTo(BeEmpty(), "the environment variable KUBECONFIG must be set")
-		kubeConfig, err = os.ReadFile(kubeConfigLocation)
+		kubeConfig, err = KubeconfigFromEnvVar("KUBECONFIG")
```

Comment:
I would propose to rename this environment variable to avoid accidents while running e2e locally but scoped against the wrong environment. With a dedicated environment variable it will be harder to misconfigure the e2e run against the wrong cluster
```suggestion
		kubeConfig, err = KubeconfigFromEnvVar("E2E_KUBECONFIG")
```